go-controller/pkg/util/gateway-init.go

```diff
@@ -395,3 +395,18 @@ func GatewayInit(clusterIPSubnet []string, nodeName, ifaceID, nicIP, nicMacAddre
 
 	return nil
 }
+
+// getNodeChassisID() will return the ChassisID from SBDB
+func getNodeChassisID(nodeName string) (string, error) {
+	chassisID, stderr, err := RunOVNSbctl("--data=bare", "--no-heading",
```

Comment:
can you please rename this to `getNodeChassisIDFromSB`? there is already GetNodeChassisID() that gets the system-id from Open_vSwitch table.